src/foam/classloader/NodeModelFileDAO.js

```diff
@@ -23,20 +23,40 @@ foam.CLASS({
   properties: [
     {
       name: 'classpath'
+    },
+    {
+      name: 'loadedModels',
```

Comment:
Ah I see.

Maybe require wasn't the best thing to use here, but rather readFile and eval/vm.runInThisContext

But I guess this is okay too.